<launch>
    <arg name="gpu" default="-1"/>
    <!--<arg name="model" default="yolo/epoch-100.model" />-->
    <group>
        <node pkg="cloud_mobilenet" args="--gpu $(arg gpu)" cwd="node" name="server" type="server.py" output="screen" />
    </group>
</launch>
